tests/shared/src/test/scala/munit/AssertionsSuite.scala

```diff
@@ -54,49 +54,50 @@ class AssertionsSuite extends BaseSuite {
     assertEquals(Option(1), Option(1))
   }
 
-  test("false-negative") {
-    assertNoDiff(
-      compileErrors("assertEquals(List(1), Vector(1))"),
-      if (isDotty)
-        """|error:
-           |Can't compare these two types:
-           |  First type:  List[Int]
-           |  Second type: Vector[Int]
-           |Possible ways to fix this error:
-           |  Alternative 1: provide an implicit instance for Compare[List[Int], Vector[Int]]
-           |  Alternative 2: upcast either type into `Any` or a shared supertype.
-           |I found:
-           |
-           |    munit.Compare.compareSubtypeWithSupertype[List[Int], Vector[Int]](
-           |      /* missing */summon[List[Int] <:< Vector[Int]]
-           |    )
-           |
-           |But no implicit values were found that match type List[Int] <:< Vector[Int].
-           |
-           |The following import might make progress towards fixing the problem:
-           |
-           |  import munit.CustomCompare.fromCustomEquality
-           |
-           |assertEquals(List(1), Vector(1))
-           |                               ^
-           |""".stripMargin
-      else
-        """|error:
-           |Can't compare these two types:
-           |  First type:  List[Int]
-           |  Second type: scala.collection.immutable.Vector[Int]
-           |Possible ways to fix this error:
-           |  Alternative 1: provide an implicit instance for Compare[List[Int], scala.collection.immutable.Vector[Int]]
-           |  Alternative 2: upcast either type into `Any` or a shared supertype
-           |assertEquals(List(1), Vector(1))
-           |            ^
-           |""".stripMargin
-    )
-  }
+  if (BuildInfo.scalaVersion != "3.3.3") {
```

Comment:
Ignored the tests specifically for 3.3.3 so this should not be ignored for the next LTS

Reply:
looks good 👍 

another possibility would be to have  in each of the test as first line
```Scala
assume(BuildInfo.scalaVersion != "3.3.3")
```

Reply:
That's a nicer solution, thanks!